src/main/java/seedu/address/model/person/Subjects.java

```diff
@@ -0,0 +1,77 @@
+package seedu.address.model.person;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.ListEntryField;
+
+/**
+ * Represents a group of Subjects in the application.
+ */
+public class Subjects extends ListEntryField {
```

Comment:
Actually, why is there a need for a group of Subjects?

Reply:
Is it because you can add multiple subjects as a field when editing / entering?

Reply:
Because before the change, subjects is a hashSet of subjects, I refractor it into Subjects type, but preserve all the original function.
I do not like the field "Subjects" myself by the way, and will have no trouble with deleting it entirely from us code.